bin/validators.js

```diff
@@ -48,6 +48,30 @@ module.exports = {
     return Boolean(firstOption && firstOption.allowIndentationTabs);
   },
 
+  "unicorn/template-indent"({ options }) {
+    if (options.length === 0) {
+      return false;
+    }
+
+    const firstOption = options[0];
+
+    if (!firstOption) {
+      return false;
+    }
+
+    const tags = firstOption.tags || [];
+    const comments = firstOption.comments || [];
+
+    return (
+      (comments.includes("HTML") ||
```

Comment:
```suggestion
      (firstOption.comments?.includes("HTML") ||
```

No need to have a temporary variable `comments`.

Reply:
Yeah, I can also remove the other empty array and refactor if you really care about performance